autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -252,7 +315,25 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             set_best_to_refit_full : bool, default = False
                 If True, will set Trainer.best_model = Trainer.full_model_dict[Trainer.best_model]
                 This will change the default model that Predictor uses for prediction when model is not specified to the refit_full version of the model that previously exhibited the highest validation score.
-                Only valid if `refit_full=True`.
+                Only valid if `refit_full` is set.
```

Comment:
I think: Only valid if `refit_full=True` was clearer.  Consider moving `refit_full` arg next to this one, so their documentation is closer together.

Reply:
refit_full can be 'all', True, 'best', 'LightGBMClassifier', etc. Saying only True at this point would be misleading.